dbms/src/Common/LRUCache.h

```diff
@@ -184,6 +184,12 @@ class LRUCache
         misses = 0;
     }
 
+    bool contains(const Key & key) const
+    {
+        std::lock_guard cache_lock(mutex);
```

Comment:
I think scoped_lock is always a better alternative than lock_guard?
see https://stackoverflow.com/questions/43019598/stdlock-guard-or-stdscoped-lock

Reply:
good point, fixed